_includes/tutorials/rekeying-function/ksql/code/tutorial-steps/dev/key-query.sql

```diff
@@ -0,0 +1,3 @@
+SELECT ROWKEY, ID, FIRSTNAME, LASTNAME, REGEXREPLACE(phonenumber, '\\(?(\\d{3}).*', '$1')
```

Comment:
If it's not too much trouble, could we preserve the phone number in this example and make a new column instead? It might be nice to see both side-by-side. Your call.

Reply:
Done